gwpy/signal/filter_design.py

```diff
@@ -306,7 +306,40 @@ def bilinear_zpk(zeros, poles, gain, fs=1.0, unit='Hz'):
     return dzeros, dpoles, dgain
 
 
-def parse_filter(args, analog=False, sample_rate=None):
+def convert_to_digital(form, filter, sample_rate):
```

Comment:
It is reasonable to call this method `bilinear`, since that's what it actually does, or is that just confusing for the user?